mmv1/third_party/terraform/resources/resource_dataproc_cluster.go.erb

```diff
@@ -680,7 +757,7 @@ func resourceDataprocCluster() *schema.Resource {
 											"cluster_config.0.preemptible_worker_config.0.disk_config",
 										},
 										ForceNew:     true,
-										ValidateFunc: validation.StringInSlice([]string{"PREEMPTIBILITY_UNSPECIFIED", "NON_PREEMPTIBLE", "PREEMPTIBLE"}, false),
+										ValidateFunc: validation.StringInSlice([]string{"PREEMPTIBILITY_UNSPECIFIED", "NON_PREEMPTIBLE", "PREEMPTIBLE", "SPOT"}, false),
```

Comment:
What are your thoughts on adding a test for `SPOT`? I think since it appears to be a new option here, it would be good to confirm it works.

Reply:
Added.